code/modules/clothing/suits/ghostsheet.dm

```diff
@@ -21,9 +21,3 @@
 	. = ..()
 	if(!isinhands && check_holidays(HALLOWEEN))
 		. += emissive_appearance('icons/mob/simple/mob.dmi', "ghost", offset_spokesman = src, alpha = src.alpha)
-
-/obj/item/clothing/suit/costume/ghost_sheet/spooky
```

Comment:
now how will we prank mappers by having them accidentally place an admin item on their map...

Reply:
look it was this or one guy murderboning everyone with incorporeality that doesn't even go away when dropped